USE-CASES/hazard-annotations.md

```diff
@@ -0,0 +1,135 @@
+## Title: Connected Hazards
+
+### Submitter(s):
+
+Luca Barbato
+
+### Reviewer(s):
+
+<Suggest reviewers>
+
+### Tracker Issue ID:
+
+<please leave blank>
+
+### Category:
+
+"vertical"
+
+### Class:
+
+<please leave blank>
+
+### Status:
+
+<please leave blank>
+
+### Target Users
+
+- device owners
+- device users
+- device manufacturers
+
+
+### Motivation:
+
+Connected devices have an impact on the physical world, more than often interacting with them may have unintended consequences.
+
+The use-case originated within [SIFIS-Home](https://sifis-home.eu) project, where the focus is on trustworthy connected homes:
+The user should be made fully aware of the hazards related to the connected devices operations, be able to declare policies regarding acceptable risks, and have the home system enforce them.
+
+The focus is on the problem of having the information available and how to deliver it to all the possible actors.
+Few scenarios on how the information could be consumed in useful and advanced ways are provided.
+
+### Expected Devices:
+
+Any device that may pose a physical or privacy hazard to the user should provide information on the risks e.g.:
+    - Fire hazards: Ovens, Stoves, Lamps
+    - Flood hazards: Windows, Faucets
+    - Invasion: Doors, Windows
+    - Privacy: Cameras, Microphones
+
+Any device that can be used to detect problems ongoing e.g.:
+    - Smoke detectors
+    - Motion detectors
+
+Any device with the ability to aggregate the information available and either prevent behavior or detect behavior.
+
+### Expected Data:
+
+The devices should provide some kind of labeling metadata to deliver the hazard information connected with all the possible interactions with it.
+
+It could be as coarse as cover any access to the device or specific to the point of selecting a range of operation within the single specific interaction, e.g.:
+- Accessing a connected camera is a privacy hazard
+- Setting an high level of brightness on an halogen lamp might involve risks of overheating it or setting something nearby on fire.
+
+Some devices may provide additional information on hazards they may simply detect and default policies other devices may try to enforce e.g.:
+- A smart meter may notify that the energy flowing through it is over a threshold and the information may trigger a policy that sets some smart switches off.
+
+Some devices may provide information that can be aggregated together to derive/evaluate a risk:
+- A CO₂/VOC sensor may provide a default policy that modulate the smoke and fire hazard evaluation based on its own information, but in itself may not detect smoke or temperature.
```

Comment:
Carbon monoxide is more hazardous than carbon dioxide and CO sensors are often combined with smoke sensors.  CO2 is more of an air quality issue (a high level indicates a need for ventilation, but unless it is *really* high is not that serious of hazard.  But even a small amount of CO is a life-threatening hazard).
```suggestion
- A CO/CO₂/VOC sensor may provide a default policy that modulate the smoke and fire hazard evaluation based on its own information, but in itself may not detect smoke or temperature.
```